src/main/kotlin/uk/gov/justice/digital/hmpps/crimeportalgateway/application/BaseTelemetryConfig.kt

```diff
@@ -1,16 +1,18 @@
 package uk.gov.justice.digital.hmpps.crimeportalgateway.application
 
-import com.microsoft.applicationinsights.boot.dependencies.apachecommons.lang3.StringUtils
 import org.springframework.context.annotation.Condition
 import org.springframework.context.annotation.ConditionContext
 import org.springframework.core.type.AnnotatedTypeMetadata
 import org.springframework.lang.NonNull
+import org.springframework.util.StringUtils
 
 open class BaseTelemetryConfig {
-
     class AppInsightKeyAbsentCondition : Condition {
-        override fun matches(@NonNull context: ConditionContext, @NonNull metadata: AnnotatedTypeMetadata): Boolean {
-            return StringUtils.isEmpty(context.environment.getProperty("application.insights.ikey"))
+        override fun matches(
+            @NonNull context: ConditionContext,
+            @NonNull metadata: AnnotatedTypeMetadata,
+        ): Boolean {
+            return !StringUtils.hasLength(context.environment.getProperty("application.insights.ikey"))
```

Comment:
This property doesn't exist, I don't think. We can probably just delete this class altogether. it looks like the spring profile is used to do the job this condition was doing